src/main/java/org/broadinstitute/consent/http/resources/DarCollectionResource.java

```diff
@@ -192,16 +190,7 @@ public Response cancelDarCollectionByCollectionId(
         actingRole = validateUserHasRoleName(user, roleName);
       }
 
-      DarCollection cancelledCollection =
-          switch (actingRole) {
-            case ADMIN -> darCollectionService.cancelDarCollectionElectionsAsAdmin(collection);
-            case CHAIRPERSON ->
-                darCollectionService.cancelDarCollectionElectionsAsChair(collection, user);
-            default -> {
-              validateUserIsCreator(user, collection);
-              yield darCollectionService.cancelDarCollectionAsResearcher(collection);
-            }
-          };
+      DarCollection cancelledCollection = darCollectionService.cancelDarCollectionElections(user, collection, actingRole);
```

Comment:
Does cancelDarCollectionElections now handle the case of the RESEARCHER turning a non-progress report DAR back into a draft?  If so, the name feels off here.

Reply:
I see that it does.  Maybe name it: cancelDarCollectionByRole?

Reply:
I've updated the name and the method comment.